freedomofpress/securedrop-client
# Basic export integration with placeholder messaging

# Description

Adds export functionality to the client.

# Test Plan

From Qubes:

1. Start the client and make sure the Export VM is running and configured correctly
2. Plug in your luks-encrypted transfer device and click Export next to the file you want to export
3. Type in your passphrase and click Submit
4. Once the transfer is successful, you will see a SecureDrop Qubes notification saying so
5. Check you transfer device for the file

Test this with the wrong password and without a transfer device plugged in. If you see any strange behavior, first check the known issues in the securedrop-export repo to see if it's already documented.

# Checklist

If these changes modify code paths involving cryptography, the opening of files in VMs or network (via the RPC service) traffic, Qubes testing in the staging environment is required. For fine tuning of the graphical user interface, testing in any environment in Qubes is required. Please check as applicable:
